pytorch_lightning/tuner/lr_finder.py

```diff
@@ -40,6 +41,8 @@
 
 def _run_lr_finder_internally(trainer, model: LightningModule):
     """ Call lr finder internally during Trainer.fit() """
+    if trainer.fast_dev_run:
+        rank_zero_warn('Skipping learning rate finder since `fast_dev_run=True`', UserWarning)
     lr_finder = lr_find(trainer, model)
     lr = lr_finder.suggestion()
```

Comment:
this will still call the `lr_find` right? Why not keep this condition under inside the functions `lr_find` since they are both calling it and the same for `scale_batch_size`?